h2o-genmodel/src/main/java/hex/genmodel/CategoricalEncodingAsModelProcessor.java

```diff
@@ -0,0 +1,33 @@
+package hex.genmodel;
+
+import hex.genmodel.easy.CategoricalEncoder;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowToRawDataConverter;
+
+import java.util.Map;
+
+public class CategoricalEncodingAsModelProcessor implements MojoPreprocessor.ModelProcessor {
```

Comment:
allows to turn `CategoricalEncoding` into a preprocessor to create a single `CompositeRowToRawDataConverter` at the end.